zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/steps/InterPartitionCommandServiceStep.java

```diff
@@ -87,23 +87,22 @@ private void waitForSenderToStart(
 
   private ActorFuture<Void> installReceiver(final PartitionTransitionContext context) {
     final ActorFuture<Void> future = context.getConcurrencyControl().createFuture();
-
+    final var logStreamWriter = context.getLogStream().newLogStreamWriter();
```

Comment:
:question: If log stream is closed, will I fail the partition entirely due to an unhandled exception?

Reply:
Same as for the `BackupApiRequestHandlerStep`: the end result is the same because this exception would be handled by `ZeebePartition#onInstallFailure`. The only difference is that we now rely on `ZeebePartition#handleFailure` instead of the error handling callback in `ZeebePartition#leaderTransition`.